test/tomo/backends/astra_setup_test.py

```diff
@@ -204,47 +204,47 @@ def test_astra_projection_geometry():
                                            det_grid)
     odl.tomo.astra_projection_geometry(geom_p2d)
 
-    # PARALLEL 2D GEOMETRY
+    # Parallel 2D geometry
     geom_p2d = odl.tomo.Parallel2dGeometry(angle_intvl, dparams, angle_grid,
                                            det_grid)
-    ageom = odl.tomo.astra_projection_geometry(geom_p2d)
-    assert ageom['type'] == 'parallel'
+    astra_geom = odl.tomo.astra_projection_geometry(geom_p2d)
+    assert astra_geom['type'] == 'parallel'
 
     # FANFLAT
     src_rad = 10
     det_rad = 5
     geom_ff = odl.tomo.FanFlatGeometry(angle_intvl, dparams, src_rad, det_rad,
                                        agrid=angle_grid, dgrid=det_grid)
-    ageom = odl.tomo.astra_projection_geometry(geom_ff)
-    assert ageom['type'] == 'fanflat_vec'
+    astra_geom = odl.tomo.astra_projection_geometry(geom_ff)
+    assert astra_geom['type'] == 'fanflat_vec'
 
     dparams = odl.IntervalProd([-40, -3], [40, 3])
     det_grid = odl.uniform_sampling(dparams, (10, 5))
 
-    # PARALLEL 3D GEOMETRY
+    # Parallel 3D geometry
     geom_p3d = odl.tomo.Parallel3dGeometry(angle_intvl, dparams, angle_grid,
                                            det_grid)
     odl.tomo.astra_projection_geometry(geom_p3d)
-    ageom = odl.tomo.astra_projection_geometry(geom_p3d)
-    assert ageom['type'] == 'parallel3d_vec'
+    astra_geom = odl.tomo.astra_projection_geometry(geom_p3d)
+    assert astra_geom['type'] == 'parallel3d_vec'
 
-    # CIRCULAR CONEFLAT
+    # Circular conebeam flat
     geom_ccf = odl.tomo.CircularConeFlatGeometry(angle_intvl, dparams, src_rad,
                                                  det_rad,
                                                  agrid=angle_grid,
                                                  dgrid=det_grid)
-    ageom = odl.tomo.astra_projection_geometry(geom_ccf)
-    assert ageom['type'] == 'cone_vec'
+    astra_geom = odl.tomo.astra_projection_geometry(geom_ccf)
+    assert astra_geom['type'] == 'cone_vec'
 
-    # HELICAL CONEFLAT
+    # Helical conebeam flat
     spiral_pitch_factor = 1
     geom_hcf = odl.tomo.HelicalConeFlatGeometry(angle_intvl, dparams, src_rad,
                                                 det_rad,
                                                 spiral_pitch_factor,
                                                 agrid=angle_grid,
                                                 dgrid=det_grid)
```

Comment:
You can write
```python
geom_hcf = odl.tomo.HelicalConeFlatGeometry(
    angle_intvl, dparams, src_rad, det_rad, spiral_pitch_factor,
    agrid=angle_grid, dgrid=det_grid)
```
which looks less squeezed (break line immediately after parenthesis and indent).